.github/workflows/fedora_29.yml

```diff
@@ -49,20 +49,26 @@ jobs:
 
   Docker:
     needs: Smart_CI
-    if: "!needs.smart_ci.outputs.skip_workflow"
     runs-on: aks-linux-4-cores-16gb-docker-build
     container:
       image: openvinogithubactions.azurecr.io/docker_build:0.2
       volumes:
         - /mount:/mount
     outputs:
-      images: "${{ steps.handle_docker.outputs.images }}"
+      images: "${{ steps.handle_docker.outputs.images && steps.handle_docker.outputs.images || steps.mock_image.outputs.images }}"
     steps:
+      - name: Set mock output images if pipeline should be skipped
```

Comment:
We can probably return this mock directly from Smart CI job to avoid copy-paste?

Reply:
If we are to return it from the `Smart_CI` job, we would need to somehow conditinally insert it in the string input for the subsequent **jobs**:
![image](https://github.com/user-attachments/assets/28eb9b40-83b1-4548-b937-5e6d4f3d0118)

which would require a lot of changes, imo.

Reply:
Ok, let's merge it as-is to unblock related PRs; we'll think of improvements later